audiofile/core/io.py

```diff
@@ -104,10 +117,18 @@ def read(
     All other audio files are
     first converted to WAV by sox or ffmpeg.
 
+    The duration values ``duration`` and ``offset``
```

Comment:
Maybe:

The arguments `duration` and `offset` support the formats mentioned in :func:`audmath.duration_in_seconds`, where `sampling_rate` is set to the sampling rate of the file, i.e. strings without a unit will be interpreted as samples.

Reply:
Changed to 

![image](https://user-images.githubusercontent.com/173624/217236092-cd6cf747-ec76-4b74-b9da-55314712ce1a.png)

Reply:
This way it sounds like a user should provide seconds, but that we internally convert internally to a seconds is just a implementation detail I think.

Reply:
> where `sampling_rate` is set to the sampling rate of the file,

This is wrong, or better we don't want to have this, because it would mean a float or integer will be treated as samples as sampling rate is always set.

I updated the docstring now to

![image](https://user-images.githubusercontent.com/173624/217291701-0e6cf582-97b1-4b07-a7c6-6de871943f81.png)

Reply:
Oh dear, I completely missed that. Of course you are right. It's a bit unfortunate we have to deal with this different behavior, but the way you phrased it now makes it clear enough I think.